SqlServerAutoTuningDashboard/Views/Home/Index.cshtml

```diff
@@ -1,37 +1,45 @@
-﻿<div class="row">
-    <div class="col-md-4">
-        <h1>Automatic tuning</h1>
-    </div>
+<div class="page-header text-left">
+  <h1>SQL Server 2017 sur Linux - "Automatic Tuning"</h1>
 </div>
 <div class="row">
-    <div class="col-md-12">
-        <span id="speed">0</span> requests per second. <button id="regression" type="button" class="btn btn-danger pull-right">Regression</button>
+    <div class="col-md-6 text-center">
+        <span><h3>Performance transactionnelle</h3></span>
     </div>
 </div>
 <div class="row">
-    <div class="col-md-6">
-        <svg width="700" height="700"></svg>
+    <div class="col-md-6 text-center">
+        <svg class="img-responsive" width="600" height="600"></svg>
+        <h4><span id="speed">0</span> requ&ecirc;tes par seconde.</h4>
+    </div>
+    <div class="col-md-6 text-left">
+        <img class="img-responsive" src="sql2017rhel74-AutoTuning-Demo-2.png" alt="">
```

Comment:
@mturcottemsft , do you want to add these sql2017rhel74-AutoTuning-Demo-xxx.png files as well in this PR?